index.js

```diff
@@ -29,20 +33,104 @@ cp.parseBlock = function () {
 	return this.finishNode(node, "BlockStatement");
 };
 
+function strToModuleIdentifier(str) {
+	return '$m_' + str.replace(/-(\w)/g, (_, letter) => letter.toUpperCase()).replace(/[^$_a-zA-Z0-9]/g, '_');
+}
+
+function writePrefix (imports) {
+	let transformed = `Object.defineProperty(exports, '__esModule', { value: true });\n`;
```

Comment:
> They will have to do this in browsers anyway 

Babel has spoiled me/us. It's tackled in some way by the Node esm proposal too.
It's also a delighter for users and smooths over a common consumption hurdle.